### PR TITLE
Allow selecting file when running -l in interactive mode

### LINKED ISSUE #165: Selecting video file
Sometimes a torrent contains multiple video files within it.  Is there a way to select which video file I can stream within a torrent, instead of just having peerflix default to the largest file?

### PR DESCRIPTION
**Pull request by clkao**
Closes #165

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -116,11 +117,32 @@ var ontorrent = function (torrent) {
...
-      })
-      process.exit(0)
+      if (interactive) {
+        inquirer.prompt([{
+          type: 'list',
+          name: 'file',
+          message: 'Choose one file',
+          choices: engine.files.map(function (file, i) {
+            return {
+              name: file.name,
```
Comment (asapach) [inline app.js]: What about file size?

Comment (clkao) [inline app.js]: now displayed as well ;)

**Comment by mafintosh** [comment]
This is pretty sweet!

**Comment by mafintosh** [comment]
Released in 0.26.0

**Pull request merged**